packages/extension-api/src/extension-api.d.ts

```diff
@@ -1149,6 +1149,22 @@ declare module '@podman-desktop/api' {
     export const onDidStateChange: Event<boolean>;
   }
 
+  export interface DefaultRegistry {
```

Comment:
suggestion(non-blocking): refactor the registy here with RegistrySuggestedProvider